services/harmony/env-defaults

```diff
@@ -470,6 +470,15 @@ GEOLOCO_REQUESTS_MEMORY=128Mi
 GEOLOCO_LIMITS_CPU=2048m
 GEOLOCO_LIMITS_MEMORY=2048Mi
 GEOLOCO_INVOCATION_ARGS='python harmony_python_interface/adapter.py'
+GEOLOCO_SERVICE_QUEUE_URLS='["ldds/geoloco:latest,http://sqs.us-west-2.localhost.localstack.cloud:4566/000000000000/geoloco.fifo"]'
+
+SUBSETNAMEBAND_IMAGE=ldds/subset_band_name:latest
```

Comment:
by convention the env variable names should match the image name, using uppercase and underscores. So `SUBSETNAMEBAND` would be `SUBSET_BAND_NAME`

Reply:
@indiejames it should see the variables/data for longitude, latitude and  EV_500_RefSB.